tools/autoreleasetagger/README.md

```diff
@@ -0,0 +1,91 @@
+# Auto Release Tagger
```

Comment:
Up top can we also specify when you would use this tool? It's during the release process? And what is this new tool affording us that we did not have previously (with the old release process)?